pkgs/by-name/li/linien-client/package.nix

```
@@ -0,0 +1,36 @@
+{ lib
+, python3
+, fetchFromGitHub
+, linien-common
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "linien-client";
+  version = "0.8.0";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "linien-org";
+    repo = "linien";
+    rev = "master";
+    hash = "sha256-JYXJEa3HerBTYzdr6AK6tWVa5auVy4REsTpO9j1d/HE=";
+  };
+
+  sourceRoot = "source/linien-client";
+
+  nativeBuildInputs = with python3.pkgs; [ setuptools ];
+
+  propagatedBuildInputs = with python3.pkgs; [
+    fabric
+    numpy
+    scipy
+    linien-common
+  ];
+
+  meta = with lib; {
+    description = "Client components of the Linien spectroscopy lock application.";
```

```suggestion
    description = "Client components of the Linien spectroscopy lock application";
```
Please remove the trailing period.
https://nixos.org/manual/nixpkgs/stable/#var-meta-description